doc/man/zcashd.1

```diff
@@ -1,9 +1,9 @@
 .\" DO NOT MODIFY THIS FILE!  It was generated by help2man 1.47.6.
```

Comment:
Shouldn't we update this to zclassicd

Reply:
updated

Reply:
shouldn't the filename be updated as well?